LemonadeGui.py

```diff
@@ -147,6 +149,32 @@ def draw_help(self):
             block = self._blit_to_block([
                 _("Weather can affect the amount of lemonade you sell."),
                 _("Watch the weather each day to see how it changes sales.")])
+        if self.help:
+            new_info = _("""
+            Help screen for Lemonade Stand
+
+            <ENTER>:
+            From Welcome Screen - Enters the game.
+            From Main Screen - Enter the store.
+            From Store Screen - Complete transaction, Leave store.
+            From Daily Log - Submits values entered.
+
+            Arrow Keys:
+            From Store Screen - Left and right arrows select a product.
+            From Daily Log - Up and down arrows select a monetary value (dollars, quarters, etc).
+
+            Number Keys:
+            From Store Screen - Used to enter the desired quantity of the selected product.
+            From Daily Log - Used to enter the number of dollars, quarters, etc. which make up your profit.
+
+            <h>:
+            From any screen - Causes a help message to appear or disappear.
+
+            <ESC>:
+            From any screen - Exits the game.
+            """)
```

Comment:
For readability, the quotes should start and end on a line by itself.

Reply:
Fixed